.env

```diff
@@ -1,13 +1,14 @@
-ADMINKEY=random3g333728eyudh3mdm3hmgkey
-AUTHKEY=fueyfhjrfjrfhvrhfgvg
-ADMIN_SIGNATURE=hdjhfjfrbbfrjb33jrjrdmb
+ADMINKEY=randomkey
```

Comment:
add this .env file to the .gitignore

Reply:
okay, I mistakenly pushed it. I will remove it now